client/network/src/discovery.rs

```diff
@@ -118,6 +120,12 @@ impl DiscoveryConfig {
 		self
 	}
 
+	/// Should the discovery behaviour randomly walk the DHT and generate ?
```

Comment:
```suggestion
	/// Whether the discovery behaviour should periodically perform a random
	/// walk on the DHT to discover peers.
```

Reply:
I tried to stick with the existing style of the other methods around, but your suggestion is indeed better.